backend/src/main/resources/db/migration/V1.22.0__NOTIF-153-email_properties.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE email_properties (
+    only_admins boolean NOT NULL,
+    ignore_preferences boolean NULL,
```

Comment:
Why is it nullable here while the Java attribute is annotated with `@NotNull`?

Reply:
because of copy/paste